js/itbit.js

```diff
@@ -160,7 +160,7 @@ module.exports = class itbit extends Exchange {
         let feeCost = this.safeFloat (trade, 'commissionPaid');
         const feeCurrencyId = this.safeString (trade, 'commissionCurrency');
         const feeCurrency = this.commonCurrencyCode (feeCurrencyId);
-        const rebatesApplied = this.safeFloat (trade, 'rebatesApplied');
+        const rebatesApplied = -(this.safeFloat (trade, 'rebatesApplied'));
```

Comment:
We need to check if it is `undefined` before trying to negate this value, otherwise it will break down in Python and PHP.